README.md

```diff
@@ -2,213 +2,209 @@
-- **Autolabeling**: Create automatic annotations
-- **Online Learning**: Simultaneously update (retrain) your model while new annotations are coming
-- **Active Learning**: Perform labeling in active learning mode - select only most complex examples
-- **Prediction Service**: Instantly create running production-ready prediction service
+- **Pre-label** your data using model predictions. 
+- **Automatically annotate** your data. 
+- Do **online learning** and retrain your model while new annotations are being created. 
+- Do **active learning** by labeling only the most complex examples in your data.
+- Set up a **prediction service** that is ready for production. 
 
-## Label Studio Integration to your services
 
-* You can use [Label Studio Frontend](https://github.com/heartexlabs/label-studio-frontend) as a separated React library, [read more here](https://labelstud.io/guide/frontend.html). 
-* Label Studio Backend (this repo) can be integrated to your app via Flask Blueprints. [See example of integration here](https://github.com/heartexlabs/label-studio/blob/master/blueprint_usage_example.py).
+## Integrate Label Studio with your machine learning tools
 
-## Label Studio for Teams, Startups, and Enterprises :office:
+You can use Label Studio as an independent part of your machine learning workflow or integrate the frontend or backend into your existing tools.  
 
-Label Studio for Teams is our enterprise edition (cloud & on-prem), that includes a data manager, high-quality baseline models, active learning, collaborators support, and more. Please visit the [website](https://www.heartex.ai/) to learn more.
+* Use the [Label Studio Frontend](https://github.com/heartexlabs/label-studio-frontend) as a separate React library. See more in the [Frontend Library documentation]](https://labelstud.io/guide/frontend.html). 
+* Use Flask Blueprints to integrate the Label Studio Backend (this repo) into your app. See [an example of this integration](https://github.com/heartexlabs/label-studio/blob/master/blueprint_usage_example.py).
 
 ## Ecosystem
 
 | Project | Description |
 |-|-|
-| label-studio | Server part, distributed as a pip package |
-| [label-studio-frontend](https://github.com/heartexlabs/label-studio-frontend) | Frontend part, written in JavaScript and React, can be embedded into your application | 
-| [label-studio-converter](https://github.com/heartexlabs/label-studio-converter) | Encode labels into the format of your favorite machine learning library | 
-| [label-studio-transformers](https://github.com/heartexlabs/label-studio-transformers) | Transformers library connected and configured for use with label studio | 
+| label-studio | Server, distributed as a pip package |
+| [label-studio-frontend](https://github.com/heartexlabs/label-studio-frontend) | React and JavaScript frontend, can run standalone in a web browser or be embedded into your application | 
```

Comment:
https://github.com/heartexlabs/dm2 is also a part of the ecosystem from now

Reply:
added